src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/DiagnosticCounter.cs

```diff
@@ -77,7 +78,7 @@ public void AddMetadata(string key, string value)
 
         public string? DisplayName { get; set; }
```

Comment:
Only somewhat related:
The ctor is validating that name isn't null, but the setter here is not.  Is that by design?  Should DisplayName also be `string` rather than `string?`?

Reply:
Changed it to be `string`. Thanks!